src/components/Character/styles.scss

```diff
@@ -1,19 +1,189 @@
+// Mobile portrait
 .wrapper {
-	font-size: 1em;
+	display: flex;
+	position: absolute;
```

Comment:
let's move this `position` down to the `top` and `left` and move that `padding` up to join the `width` and `height`

Reply:
😱